plugins/modules/azure_rm_containerregistry_tag.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2022 Ross Bender (@l3ender)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_containerregistry_tag
+version_added: "1.12.0"
+short_description: Delete tags in Azure Container Registry
+description:
+    - Delete tags in Azure Container Registry.
+
+options:
+    resource_group:
+        description:
+            - The resource group of the registry.
+        type: str
+    registry:
+        description:
+            - The name of the container registry.
+        type: str
+        required: true
+    repository_name:
+        description:
+            - The name of the repository within the registry.
+            - Required when state = C(absent).
+            - If omitted when state = C(present), the name of the source repository will be used.
+        type: str
+    name:
+        description:
+            - The name of the tag.
+            - If omitted when state = C(present), the name of the source tag will be used.
+            - If omitted when state = C(absent), the whole repository will be deleted.
+        type: str
+    source:
+        description:
+            - The source image detail. Required when state = C(present).
+        type: dict
+        suboptions:
+            registry_uri:
+                description:
+                    - The address of the source registry.
+                type: str
+            repository:
+                description:
+                    - Repository name of the source image.
+                type: str
+                required: true
+            name:
+                description:
+                    - Name of the tag.
+                type: str
+                default: latest
+            credentials:
+                description:
+                    - Credentials for the source registry.
+                type: dict
+                suboptions:
+                    username:
+                        description:
+                            - Username for the source registry
```

Comment:
All descriptions should end with a "." except shot_description!

```suggestion
                            - Username for the source registry.
```